src/adaptive.jl

```diff
@@ -0,0 +1,93 @@
+mutable struct Adaptor
+    accepted::Integer
+    total   ::Integer
+    tune    ::Integer   # tuning interval
+    target  ::Float64   # target acceptance rate
+    bound   ::Float64   # bound on logσ of Gaussian kernel
+    δmax    ::Float64   # maximum adaptation step
+end
+
+Adaptor(; tune=25, target=0.44, bound=10., δmax=0.2) =
+    Adaptor(0, 0, tune, target, bound, δmax)
+
+"""
+    AdaptiveProposal{P} 
+
+An adaptive Metropolis-Hastings proposal. In order for this to work, the
+proposal kernel should implement the `adapted(proposal, δ)` method, where `δ`
+is the increment/decrement applied to the scale of the proposal distribution
+during adaptation (e.g. for a Normal distribution the scale is `log(σ)`, so
+that after adaptation the proposal is `Normal(0, exp(log(σ) + δ))`).
```

Comment:
Can you briefly describe the default `Adaptor` in the docstring for `AdaptiveProposal`, and notify everyone what the default keywords are for the adaptor?